web/packages/teleterm/src/services/tshd/fixtures/mocks.ts

```diff
@@ -43,7 +42,6 @@ import {
 export class MockTshClient implements TshClient {
   listRootClusters: () => Promise<Cluster[]>;
   listLeafClusters: (clusterUri: string) => Promise<Cluster[]>;
-  listApps: (clusterUri: string) => Promise<Application[]>;
```

Comment:
You can also remove `getAllServers`, `getAllDatabases`, `getAllServers` and `listServers`.

Reply:
Thanks, too bad we cannot make TypeScript inform us about this here. There was this package for mocking classes automatically which was supposed to be added to jest directly but idk what happened with that.